backend/user-service/src/db/dbConfig.ts

```diff
@@ -1,7 +1,18 @@
 import { Sequelize } from 'sequelize-typescript';
 
+/*
+#For connecting to elephant sql cloud db
 const sequelize = new Sequelize(process.env.POSTGRESURL, {
   dialect: 'postgres',
 });
+*/
+
+const sequelize = new Sequelize({
+  database: process.env.PG_DB,
+  username: process.env.PG_USER,
+  password: process.env.PG_PASSWORD,
+  host: process.env.PG_HOST, // This should match the service name in your Docker Compose file
+  dialect: 'postgres',
+});
```

Comment:
Did some testing:
- This localises the postgres db instance, meaning we can eliminate the need for elephantDB's cloud instance.
- Data also persists in the docker image itself.

GJ 👍